Source/ThirdParty/libwebrtc/Configurations/Base.xcconfig

```diff
@@ -72,7 +72,7 @@ GCC_WARN_UNUSED_FUNCTION = YES;
 GCC_WARN_UNUSED_VARIABLE = YES;
 CLANG_WARN_SUSPICIOUS_MOVE = YES;
 PREBINDING = NO;
-WARNING_CFLAGS = $(inherited) $(WK_FIXME_WARNING_CFLAGS) -Wexit-time-destructors -Wglobal-constructors -Wthread-safety;
+WARNING_CFLAGS = $(inherited) $(WK_FIXME_WARNING_CFLAGS) -Wexit-time-destructors -Wglobal-constructors -Wthread-safety -Wno-unused-but-set-variable;
```

Comment:
This `-Wno-unused-but-set-variable` needs to go in `WK_FIXME_WARNING_CFLAGS` below.